src/renderer/components/wallet/txs/table/TxsTable.tsx

```diff
@@ -23,12 +25,31 @@ type Props = {
   txsPageRD: TxsPageRD
   clickTxLinkHandler: (txHash: string) => void
   changePaginationHandler: (page: number) => void
+  network: O.Option<Network>
+  asset: O.Option<Asset>
 }
 
 export const TxsTable: React.FC<Props> = (props): JSX.Element => {
-  const { txsPageRD, clickTxLinkHandler, changePaginationHandler } = props
+  const { txsPageRD, clickTxLinkHandler, changePaginationHandler, network: oNetwork, asset: oAsset } = props
   const intl = useIntl()
   const isDesktopView = Grid.useBreakpoint()?.lg ?? false
+  const network = useMemo(
+    () =>
+      FP.pipe(
+        oNetwork,
+        O.getOrElse(() => '')
+      ),
+    [oNetwork]
+  )
+  const chain = useMemo(
```

Comment:
What's type of `chain`? It seems like `Chain | string`, but you want to have `Chain` in `AddressEllipsis`, right?